tests/slow/src/test/scala/tests/best-effort/BestEffortCompilationSuite.scala

```diff
@@ -0,0 +1,299 @@
+package tests.`best-effort`
+
+import tests.BaseCompletionLspSuite
+
+class BestEffortCompilationSuite
+    extends BaseCompletionLspSuite("best-effort-compilation") {
+  val scalaVersion = "3.5.0-RC1"
+
+  // implemented in bloop
+  test("best-effort-error-diagnostics") {
```

Comment:
I wonder if we could reuse some of the tests in CompilersLspSuite for outline? We would need to make it into a base suite and extends here

Reply:
We can but error messages are different in Scala 2 and 3, I'll try to figure something out for that

Reply:
Wait nvm there are no error diagnostics in outline, I misread the logs as always...

Reply:
Done, I did have to have them test different diagnostics for scala 3 and I also had to add saving after changes, before testing completions (but for scala-2s outline they should work identically). I could not make one of the test work (not sure why), so I moved it out of the base suite to an outline only one.